nixos/modules/services/hardware/nvidia-container-toolkit/default.nix

```
@@ -52,6 +52,15 @@
         '';
       };
 
+      device-name-strategy = lib.mkOption {
+        default = [ "index" ];
```

```suggestion
        default = "index";
```

Shouldn't the default be provided like so?

---

yes, thanks for catching that one :)